jdi-light-vuetify-tests/src/test/java/io/github/epam/vuetify/tests/complex/bars/SystemBarsTests.java

```diff
@@ -1,21 +1,23 @@
 package io.github.epam.vuetify.tests.complex.bars;
 
-import static com.jdiai.tools.Timer.waitCondition;
-import static io.github.com.StaticSite.systemBarsPage;
-import static io.github.com.enums.Colors.GREEN;
-import static io.github.com.pages.SystemBarsPage.windowSystemBar;
-import static io.github.com.pages.SystemBarsPage.coloredSystemBars;
-import static io.github.com.pages.SystemBarsPage.appSystemBar;
-import static io.github.com.pages.SystemBarsPage.redColoredSystemBar;
-
-
-
 import io.github.epam.TestsInit;
 import io.github.epam.vuetify.tests.data.SystemBarDataProvider;
 import org.hamcrest.Matchers;
 import org.testng.annotations.BeforeClass;
 import org.testng.annotations.Test;
 
+import static com.jdiai.tools.Timer.waitCondition;
+import static io.github.com.StaticSite.systemBarsPage;
+import static io.github.com.enums.Colors.GREEN;
+import static io.github.com.pages.SystemBarsPage.absolutePositionSystemBar;
+import static io.github.com.pages.SystemBarsPage.appSystemBar;
+import static io.github.com.pages.SystemBarsPage.coloredSystemBars;
+import static io.github.com.pages.SystemBarsPage.darkLightsOutSystemBar;
+import static io.github.com.pages.SystemBarsPage.highSystemBar;
+import static io.github.com.pages.SystemBarsPage.lightLightsOutSystemBar;
+import static io.github.com.pages.SystemBarsPage.redColoredSystemBar;
+import static io.github.com.pages.SystemBarsPage.windowSystemBar;
+
 public class SystemBarsTests extends TestsInit {
```

Comment:
Please also refactor following tests:
- coloredSystemBarTests:
-- rename coloredSystemBarTests -> colorSystemBarTest
-- delete text check
-- BasicBarAssert and BasicBar should implement ColorAssert and HasColor
-- delete color method from BasicBarAssert
-- change color check for backgroundColor

- systemBarColoredPrimaryTests
-- there are a lot of duplicated checks -> please delete it
-- is heartButton() common element for all bars?
-- is verticalDotsButton() common element for all bars?

- windowSystemBarTests 
-- do we really need to do check icons?

- iconBarTests
-- do we need to check icons wifiIcon()/signalIcon()/batteryIcon()/timeSpan() in system bar test?